admin/spec/service/date.service.spec.js

```diff
@@ -17,4 +17,11 @@ describe('date service', () => {
       expect(dateService.formatShortGdsDate(date)).toBe('31 Dec 2010')
     })
   })
+
+  describe('#formatDayAndDate', () => {
+    it('correctly formats a date', () => {
+      const date = new Date(2010, 11, 31, 14, 10, 0, 0)
+      expect(dateService.formatDayAndDate(date)).toBe('Friday 31 December')
```

Comment:
This function takes a date but a date is not passed in from the school controller and there is no test for that usage.   Should there be?

Reply:
moment is handling that for me though, which means that I will be essentially testing the library if I add a case where there's a null date and I would expect to receive the current one in the specified format.
@jon-shipley thoughts?

Reply:
Just pass new Date() from the controller as the arg?

Reply:
https://www.sitepoint.com/functional-programming-pure-functions/

Reply:
@jon-shipley done